app/javascript/mastodon/features/ui/util/async-components.js

```diff
@@ -102,6 +102,10 @@ export function ReportModal () {
   return import(/* webpackChunkName: "modals/report_modal" */'../components/report_modal');
 }
 
+export function ActionsModal () {
+  return import(/* webpackChunkName: "modals/actions_modal" */'../components/actions_modal');
```

Comment:
Should we have this as separate chunk? Since it's only 2KB in plain and 945 bytes in gzipped, I think it's better that include it in application pack to avoid loading delay and showing empty modal, though this may also apply to other modals.